apps/backend/src/application/usecases/adopter/find-nearest-pets/find-nearest-pets.ts

```diff
@@ -0,0 +1,29 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { UserService } from '../../../../user/user.service';
+import { UserRepository } from '../../../../user/repositories/user-repository.interface';
```

Comment:
What do you think about using path aliases and barrel files to simplify and better organize imports?

Reply:
i think it improves readability and helps to better organize the imports. i'll leave it up to someone looking for a good first issue